### ISSUE TITLE
Compatibility with g++ 4.8 (older libstdc++)

### ISSUE THREAD

**Issue by springmeyer** [mapbox/wagyu#78]
I've noticed that there is only one line of code in wagyu that prevents compilation with g++-4.8. Therefore this I think is worth fixing, if only with an #ifdef. it looks like the libstdc++ that ships with g++-4.8 does not have the signature that supports `std::initializer_list`. @flippmoke can you think of a clean way to avoid this usage?

```
./deps/wagyu/include/mapbox/geometry/wagyu/active_bound_list.hpp:115:12: error: no viable conversion from returned value of type 'void' to function return type
      'active_bound_list_itr<long>' (aka '__gnu_cxx::__normal_iterator<mapbox::geometry::wagyu::bound<long> **, std::vector<mapbox::geometry::wagyu::bound<long> *,
      std::allocator<mapbox::geometry::wagyu::bound<long> *> > >')
    return active_bounds.insert(itr, { &left, &right });
```

Entire error at: https://gist.github.com/springmeyer/c89c98abfaaf6f7d3ad082fbdbb3a01d

Note: I've replicated the problem on travis and attempted to fix, but failed: https://github.com/mapbox/wagyu/compare/g++-4.8

**Comment by joto** [comment]
There is a working solution in https://github.com/mapbox/wagyu/pull/79 . At least the build now fails somewhere else. And I am trusting the tests here, don't know whether this is actually doing the right thing.

**Comment by springmeyer** [comment]
This was solved by #79. Thank you @joto.
